src/compiler/binder.ts

```diff
@@ -445,6 +445,14 @@ function getModuleInstanceStateForAliasTarget(specifier: ExportSpecifier, visite
                     if (found === ModuleInstanceState.Instantiated) {
                         return found;
                     }
+                    if (statement.kind === SyntaxKind.ImportEqualsDeclaration) {
+                        // Treat re-exports of import aliases as instantiated,

```

Comment:
this doesn't break a lot of existing tests? I remember looking at this or similar problem a few years ago and it looked very complex. *however*, I may have been making it more complicated than it needed to be.

Reply:
Maybe you were attempting to resolve whether the alias actually necessitates instantiating the namespace instead of doing it unconditionally? I don’t know; I was surprised this didn’t change existing baselines too.